mqboost/objective.py

```diff
@@ -269,6 +312,11 @@ def __init__(
             self._fobj = partial(
                 phuber_loss_grad_hess, alphas=alphas, delta=self._delta
             )
+        elif objective == ObjectiveName.phuber2:
+            self._delta = delta_validate(delta=delta)
```

Comment:
I think this is unnecessary  for your objective function, so can you delete this line please?